qldbdriver/qldbdriver.go

```diff
@@ -41,20 +40,22 @@ type DriverOptions struct {
 type QLDBDriver struct {
 	ledgerName                string
 	qldbSession               qldbsessioniface.QLDBSessionAPI
-	retryLimit                uint8
-	maxConcurrentTransactions uint16
+	retryLimit                int
+	maxConcurrentTransactions int
 	logger                    *qldbLogger
 	isClosed                  bool
 	semaphore                 *sync2.Semaphore
 	sessionPool               chan *session
+	retryPolicy               RetryPolicy
 }
 
 // New creates a QLBDDriver using the parameters and options, and verifies the configuration.
 //
 // Note that qldbSession.Client.Config.MaxRetries will be set to 0. This property should not be modified.
 // DriverOptions.RetryLimit is unrelated to this property, but should be used if it is desired to modify the amount of retires for statement executions.
 func New(ledgerName string, qldbSession *qldbsession.QLDBSession, fns ...func(*DriverOptions)) *QLDBDriver {
-	options := &DriverOptions{4, 50, defaultLogger{}, LogInfo}
+	retryPolicy := RetryPolicy{MaxRetryLimit: 4, Backoff: ExponentialBackoffStrategy{SleepBase: 10000000, SleepCap: 5000000000}}
```

Comment:
To clarify what I meant earlier, you need to say SleepBase: 10000 * Time.Duration.Milliseconds in order to specify what's happening here.